### DISCUSSION TITLE
Login screen doesn't redirect after succesfull login

### DISCUSSION THREAD

**Discussion by Muratcol** [orchidsoftware/platform#1753] [Q&A] [answered]
After logging into the platform, I am not redirected to the home page. I have to refresh the page to access the main page.

I am getting this error message.

<img width="248" alt="Untitled2" src="https://user-images.githubusercontent.com/58704132/120323679-69533d80-c2ee-11eb-8268-227e86328f0a.png">

And screen freeze like this:

<img width="495" alt="Untitled" src="https://user-images.githubusercontent.com/58704132/120323670-66f0e380-c2ee-11eb-84dd-1fd94e43d044.png">

 This problem is only exists in the production environment. There is no such problem in the local environment. Does anyone know your reason?

**Comment by tabuna** [comment, marked as answer]
The page to which the redirect was made did not respond with a 200 error.  You can see the request in the browser inspector or application logs

**Reply by Muratcol** [reply]
Ah yes, 
```
Mixed Content: The page at 'https://dashboard-prod.app/admin/login' was loaded over HTTPS, but requested an
insecure XMLHttpRequest endpoint 'http://dashboard-prod.app/admin'. This request has been blocked; the content
must be served over HTTPS.
```
Why is the request converted from https to http?

**Reply by tabuna** [reply]
We rely entirely on Laravel to define HTTPS.
 Most likely, you have a common problem when the web server does not set a value.

```php
$ _SERVER ['HTTPS']; // It should be 'on'
```

Check this value is set, as well as the setting for the webserver.

**Reply by Muratcol** [reply]
Thanks a lot @tabuna ! 

The last couple of days I was trying to fix the problem.

And finally, I managed to fix the problem like this: (Maybe someone else may run into a similar problem.)

```
providers/AppServiceProvider.php

    /**
     * Bootstrap any application services.
     *
     * @return void
     */
    public function boot()
    {
        if ($this->app->environment() !== 'local') {
            $this->app['request']->server->set('HTTPS', true);
        }
    }
```

**Reply by tabuna** [reply]
I'm glad you solved the problem.  But I still recommend changing the web server settings so that the symfony component responsible for defining https works correctly.
